### DISCUSSION TITLE
Ignore SSL Errors

### DISCUSSION THREAD

**Discussion by los93sol** [nginx-proxy/acme-companion#926] [Ideas]
I have a scenario where I have a custom ACME server with an SSL error.  The certificate is actually valid but under certain debug conditions I am pointing at an address that is not present in the certificate, hence the error.  It would be nice if there was a flag to run acme.sh with —insecure to ignore the errors.

**Comment by buchdag** [comment]
Have you tried the `CA_BUNDLE` environment variable ? I added it for test purpose but it might useful to your use case.

https://github.com/nginx-proxy/acme-companion/blob/main/docs/Container-configuration.md

**Reply by los93sol** [reply]
Thanks, that looks like what I need!
